cs/di-defaultextensions.texy

```diff
@@ -0,0 +1,86 @@
+DI: Defaultní rozšíření pro kontejner
+*************************************
+
+Nette obsahuje několik předdefinovaných [Nette\DI\CompilerExtension |api:] pro DI kontejner.
+
+ExtensionsExtension
+===================
+
+Díky tomuto rozšíření můžeme zaregistrovat další rozšiření pomocí konfiguračních NEON souborů.
+
+/---neon
+extensions:
+	blog: App\DI\BlogExtension
```

Comment:
Asi by to chtělo zmínku, že rozšíření se pak konfigurují přes to `blog` resp `user`. Tak co tam dát reálný příklad? 

```
extensions:
	dibi: Dibi\Bridges\Nette\DibiExtension22

dibi:
	host: localhost
```